### PR TITLE
PUBDEV-4248: Add a parameter that ignores the config file reader when h2o.init() is called

### PR DESCRIPTION
**Pull request by navdeep-G**
* Please see JIRA for description

### CODE REVIEW

File: h2o-py/h2o/h2o.py [hunk 1]
```diff
@@ -141,7 +141,7 @@ def version_check():
 
 def init(url=None, ip=None, port=None, https=None, insecure=None, username=None, password=None,
          cookies=None, proxy=None, start_h2o=True, nthreads=-1, ice_root=None, enable_assertions=True,
-         max_mem_size=None, min_mem_size=None, strict_version_check=None, **kwargs):
+         max_mem_size=None, min_mem_size=None, strict_version_check=None,ignore_config=False, **kwargs):
```
Comment (mmalohlava) [inline h2o-py/h2o/h2o.py]: Minor: formatting - space after `,`

File: h2o-py/h2o/h2o.py [hunk 2]
```diff
@@ -161,6 +161,7 @@ def init(url=None, ip=None, port=None, https=None, insecure=None, username=None,
     :param max_mem_size: Maximum memory to use for the new h2o server.
     :param min_mem_size: Minimum memory to use for the new h2o server.
     :param strict_version_check: If True, an error will be raised if the client and server versions don't match.
+    :param ignore_config: Indicates whether a search for a .h2oconfig file should be conducted or not. Default value is False.
```
Comment (mmalohlava) [inline h2o-py/h2o/h2o.py]: s/search for/processing of/

File: h2o-py/h2o/h2o.py [hunk 3]
```diff
@@ -210,25 +211,26 @@ def get_mem_size(mmint, mmgb):
     check_version = True
     verify_ssl_certificates = True
 
-    # Apply the config file
-    config = H2OConfigReader.get_config()
-    if url is None and ip is None and port is None and https is None and "init.url" in config:
+    # Apply the config file if ignore_config=False
+    if not ignore_config:
+        config = H2OConfigReader.get_config()
+    if not ignore_config and url is None and ip is None and port is None and https is None and "init.url" in config:
```
Comment (mmalohlava) [inline h2o-py/h2o/h2o.py]: You can move the check for `ignore_config` into a dedicated block:

```
if not ignore_config:
   if proxy is None and "init.proxy" in config:
    ....
...
```

Comment (mmalohlava) [inline h2o-py/h2o/h2o.py]: The same way as you did for R.

Comment (navdeep-G) [inline h2o-py/h2o/h2o.py]: Yep, will do so.

File: h2o-py/h2o/h2o.py (hunk 2 above)
Comment (navdeep-G) [inline h2o-py/h2o/h2o.py]: ?

File: h2o-py/h2o/h2o.py (hunk 1 above)
Comment (navdeep-G) [inline h2o-py/h2o/h2o.py]: Sounds good.

File: h2o-py/h2o/h2o.py (hunk 2 above)
Comment (mmalohlava) [inline h2o-py/h2o/h2o.py]: :D prefix by `sed -e `

**Comment by navdeep-G** [comment]
@mmalohlava Have updated via your review.

**Pull request merged**